[7.7.x] RHPAM-1107: Selected container gets changed

nmirasch opened the pull request:
ContainerPresenter reacting only on ContainerSpecSelected event when it is the one that is being displayed
Use container.getContainerSpecId() instead container.getContainerName() to map containers
clear status notifications before add new one, added by update actions

nmirasch commented:
@cristianonicolai could you take a look? Thanks!

cristianonicolai commented on kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/java/org/kie/workbench/common/screens/server/management/client/container/ContainerPresenter.java:
```diff
@@ -72,6 +72,7 @@
     private final Event<ServerTemplateSelected> serverTemplateSelectedEvent;
     private final Event<NotificationEvent> notification;
     private ContainerSpec containerSpec;
+    private ContainerSpecKey selectedContainerSpecKey;
```
@nmirasch would the _containerSpecKey_ be different from the _containerSpec_ we already hold as variable?

nmirasch commented:
@cristianonicolai agree, the have to hold the same data, the only thing that differs is when they are set. I removed selectedContainerSpecKey  and added the setting of the  variable containerSpec at load(final ContainerSpecKey containerSpecKey) to be able to use that from loadContainers(@Observes final ContainerSpecData content) .

nmirasch commented:
jenkins please retest this

cristianonicolai commented:
jenkins please retest this
